app/network_policies/bandwidthpolicydetailsctrl.ts

```diff
@@ -36,6 +36,8 @@ export class BandwidthPolicyDetailsComponent {
         /* Get particular Profile for based on key*/
         bandwidthPolicyDetailsCtrl.netprofilesModel.getModelByKey(activatedRoute.snapshot.params['key'],false,undefined)
             .then(function (policy) {
+                if (policy.DSCP === undefined) policy.DSCP = 0;
```

Comment:
How are policy.DSCP and policy.burst undefined at this point? Shouldn't they be null after parsing the JSON response?